helpick-service/src/main/java/com/infoshare/domain/NeedRequest.java

```diff
@@ -2,93 +2,96 @@
+  }
 
-    public void setTypeOfHelp(TypeOfHelp typeOfHelp) {
-        this.typeOfHelp = typeOfHelp;
-    }
+  public void setTypeOfHelp(TypeOfHelp typeOfHelp) {
+    this.typeOfHelp = typeOfHelp;
+  }
 
-    public HelpStatuses getHelpStatus() {
-        return helpStatus;
-    }
+  public HelpStatuses getHelpStatus() {
+    return helpStatus;
+  }
 
-    public void setHelpStatus(HelpStatuses helpStatus) {
-        this.helpStatus = helpStatus;
-    }
+  public void setHelpStatus(HelpStatuses helpStatus) {
+    this.helpStatus = helpStatus;
+  }
 
-    public Date getStatusChange() {
-        return statusChange;
-    }
+  public Date getStatusChange() {
+    return statusChange;
+  }
 
-    public void setStatusChange(Date statusChange) {
-        this.statusChange = statusChange;
-    }
+  public void setStatusChange(Date statusChange) {
+    this.statusChange = statusChange;
+  }
 
-    public PersonInNeed getPersonInNeed() {
-        return personInNeed;
-    }
+  public PersonInNeed getPersonInNeed() {
+    return personInNeed;
+  }
 
-    public void setPersonInNeed(PersonInNeed personInNeed) {
-        this.personInNeed = personInNeed;
-    }
+  public void setPersonInNeed(PersonInNeed personInNeed) {
+    this.personInNeed = personInNeed;
+  }
 
-    @Override
-    public String toString() {
-        return "NeedRequest{" + "typeOfHelp=" + typeOfHelp + ", helpStatus=" + helpStatus + ", statusChange="
-            + statusChange + ", personInNeed=" + personInNeed + '}';
-    }
+  @Override
+  public String toString() {
+    return "NeedRequest{" + "typeOfHelp=" + typeOfHelp + ", helpStatus=" + helpStatus + ", statusChange="
+        + statusChange + ", personInNeed=" + personInNeed + '}';
+  }
 
 
-    @Override
-    public boolean equals(Object o) {
-        if (this == o) {
-            return true;
-        }
-        if (o == null || getClass() != o.getClass()) {
-            return false;
-        }
-        NeedRequest that = (NeedRequest) o;
-        return typeOfHelp == that.typeOfHelp && helpStatus == that.helpStatus && statusChange
-            .equals(that.statusChange) && personInNeed.equals(that.personInNeed);
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
     }
-
-    @Override
-    public int hashCode() {
-        return Objects.hash(typeOfHelp, helpStatus, statusChange, personInNeed);
-    }
-
-    public void printDescription() {
-        System.out
-            .println("Imię osoby potrzebującej: " + personInNeed.getName() + ", miasto: " + personInNeed.getLocation()
-                + ", telefon: " + personInNeed.getPhone() + ", rodzaj potrzebnej pomocy: " + typeOfHelp.getType());
+    if (o == null || getClass() != o.getClass()) {
+      return false;
     }
+    NeedRequest that = (NeedRequest) o;
+    return typeOfHelp == that.typeOfHelp && helpStatus == that.helpStatus && statusChange
+        .equals(that.statusChange) && personInNeed.equals(that.personInNeed);
```

Comment:
Warto na koniec zawsze poświęcić chwilę i sformatować tak kod, żeby był bardziej czytelny 😉 
Np.:
```suggestion
    return typeOfHelp == that.typeOfHelp
        && helpStatus == that.helpStatus
        && statusChange.equals(that.statusChange)
        && personInNeed.equals(that.personInNeed);
```